### PR TITLE
SE-632-Upgrade-protocol-and-handlers-on-testnet

### PR DESCRIPTION
**Pull request by ShaneDuncan602**

### CODE REVIEW

File: script/python/get_latest_facet_address.py [hunk 1]
```diff
@@ -43,6 +43,10 @@ def main():
         print("Not Found")
         return
     enc = encode(["address[2]"], [[facet, diamond]])
+    # enc = encode(["(address,address)"], [(facet, diamond)])
+    # enc = encode(["address"], [facet])
+    # print(enc)
+    # print("0x01462DcB0DE6ea6b9Aee343C6c1280A17259969a","0x826149518A0D4Cf67b58fB8698b478CeaE96708E")
```
Comment (Palmerg4) [inline script/python/get_latest_facet_address.py]: Are these comments permanent?

File: docs/userGuides/rules/README.md [hunk 2]
```diff
@@ -32,23 +32,6 @@ Rule application is a two step process. First, the rule admin must obtain a rule
...
-| [Token Minimum Hold Time](./TOKEN-MIN-HOLD-TIME.md) | The purpose of the token-min-hold-time rule is to reduce trade volitility by preventing transfers of tokens for a number of hours after ownership is acquired, either via minting or transfers. This rule allows developers to specifiy a number of hours, up to 43830 (5 years), that each tokenId must be held for. |
-| [Token Minimum Transaction](./TOKEN-MIN-TRANSACTION-SIZE.md)| The purpose of the token-min-tx-size rule is to prevent micro-trades or dust trades within an ecosystem. A developer can set the minimum number of tokens required per transfer to prevent these types of trades. |
```
Comment (Palmerg4) [inline docs/userGuides/rules/README.md:51]: Are these definitions going to be relocated or purged? Do they need to be updated?

File: script/python/get_latest_facet_address.py (hunk 1 above)
Comment (ShaneDuncan602) [reply script/python/get_latest_facet_address.py]: Good catch. These should be deleted.

File: docs/userGuides/rules/README.md (hunk 2 above)
Comment (ShaneDuncan602) [reply docs/userGuides/rules/README.md:51]: These were duplicates that already exist at the top of the README. Somehow they must've been copy/pasted twice so I deleted the redundant section.

**Pull request merged**